addons/account_peppol/models/account_edi_proxy_user.py

```diff
@@ -76,6 +76,55 @@ def _get_proxy_identification(self, company, proxy_type):
             return f'{company.peppol_eas}:{company.peppol_endpoint}'
         return super()._get_proxy_identification(company, proxy_type)
 
+    def _peppol_save_new_document(self, attachment, company, partner_endpoint, peppol_state, uuid):
+        """Save new documents in an accounting journal, when one is specified on the company.
+
+        :param attachment: the new document
+        :param company: the company where we want to save the document
+        :param partner_endpoint: a string containing the sender's Peppol endpoint
+        :param peppol_state: the state of the received Peppol document
+        :param uuid: the UUID of the Peppol document
+        :return: `True` if the document was saved, `False` if it was not
+        """
+        journal_id = company.peppol_purchase_journal_id
```

Comment:
> Was a bug before this commit to not take `peppol_purchase_journal_id`?

I'm not entirely sure what you mean. Before, we only had the `peppol_purchase_journal_id` and always needed to save the document in a journal (even if it happened to be not specified).
Now we can either save it in a journal (when it is set), in Documents (when the workspace is set), or in both when both are set.
If it would happen that none are set, I don't think we can default to something then (since we don't know whether the document is saved in the other method).

> Also, for the sake of consistency: `record_id` is for an **ID** so an integer!
> Rename it to `journal`.

This was just moved from the other method, but I'll change it.

Reply:
`journal_id` was changed to `journal`